### DISCUSSION TITLE
Using this Plugin to Integrate with Blue Iris NVR server

### DISCUSSION THREAD

**Discussion by radensb** [homebridge-simplisafe3/homebridge-simplisafe3#212] [Q&A]
Thanks for all the work that was put into this! Cheers to everyone involved. I went from having never heard of Homebridge to having it up and running with the SS3 plugin in about 15.5 minutes. My goal right now doesnt involve using it with HomeKit, but rather as a standalone monitor for the Simplisafe status. I am running BlueIris NVR on the same VM that Homebridge is running and I wanted to be able to send commands to BlueIris based on the state of the SimpliSafe system. IE:

Alarm set to OFF -> BlueIris switches to OFF profile
Alarm set to HOME -> BlueIris switches to Home profile
Alarm set to AWAY -> BlueIris switches to Away profile
Alarm triggers -> BlueIris switches to Emergency profile

Im very new with Homebridge and feel like I am missing something to accomplish this integration. I wanted to write a script that gets called on alarm state changes so that the proper command can be sent to the BlueIris server to set the appropriate profile. This seems like it should be easily accomplished since all of the system state information is available with the plugin, but I am unsure how to proceed. Im not even sure if this is this a plugin related question, or more of a Homebridge host question. Any guidance would be great.

Thanks for the great work!

**Comment by shamoon** [comment]
Hi @radensb , thanks for the kind words. Yea I think what youre asking about is outside the scope of this plugin. I dont know specifically about Blueiris but with SimpliSafe in Homebridge you need a way to create an automation (in HomeKit) that triggers your NVR system. So either that system has a homebridge plugin of its own (doesnt seem like if after a quick Google search) or you can find a way to trigger it with another plugin (e.g. there are plugins that can trigger HTTP call or something like that). There are other workaround methods too (e.g. https://www.pushcut.io ). Or you may be looking for something a bit more complex (and nothing to do with Homebridge, btw) like https://www.home-assistant.io

Good luck, of course feel free to report back if you think it might be helpful for others.
